### PR TITLE
Add documentation for iCub as a wearable device

### PR DESCRIPTION
**Pull request by lrapetti**
Minor changes:
- `/conf` folder is removed since configuration files are stored in `/app`
- minor improvement of existing documentation

### CODE REVIEW

**Comment by lrapetti** [comment]
The CI checks failed with
```
The job exceeded the maximum time limit for jobs, and has been terminated
```
at the MacOs `clang` step.

Trying to run checks again.

File: doc/How-to-run-iCub-as-wearable-source.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+**Before running**
+Before running the device make sure that:
+- [`yarpserver`](https://www.yarp.it/yarpserver.html) is running
+- The `YARP_ROBOT_NAME` is set to the proper robot name. 
+- The robot is currently running (either [Gazebo simulator](https://github.com/robotology/icub-gazebo) or real robot making sure it is runned with the [proper configuration for whole-body-dynamics](https://github.com/robotology/whole-body-controllers/blob/master/doc/How-to-setup-the-robot-for-wbc-experiments.md))
+- If the robot is running in simulation, the `whole-body-dynamics` estimatior has to be launched
+  ```
+  yarprobotinterface --config launch-wholebodydynamics.xml
+  ```
+  It the real robot is running, `whole-body-dynamics` estimatior should be running if the robot is launched with the [proper configuration](https://github.com/robotology/whole-body-controllers/blob/master/doc/How-to-setup-the-robot-for-wbc-experiments.md).
```
Comment (Yeshasvitvs) [inline doc/How-to-run-iCub-as-wearable-source.md]: @lrapetti small typo **It** the real robot is running ---------> **If**

Comment (Yeshasvitvs) [reply doc/How-to-run-iCub-as-wearable-source.md]: or When

File: doc/How-to-run-iCub-as-wearable-source.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+Before running the device make sure that:
+- [`yarpserver`](https://www.yarp.it/yarpserver.html) is running
+- The `YARP_ROBOT_NAME` is set to the proper robot name. 
+- The robot is currently running (either [Gazebo simulator](https://github.com/robotology/icub-gazebo) or real robot making sure it is runned with the [proper configuration for whole-body-dynamics](https://github.com/robotology/whole-body-controllers/blob/master/doc/How-to-setup-the-robot-for-wbc-experiments.md))
+- If the robot is running in simulation, the `whole-body-dynamics` estimatior has to be launched
+  ```
+  yarprobotinterface --config launch-wholebodydynamics.xml
+  ```
+  It the real robot is running, `whole-body-dynamics` estimatior should be running if the robot is launched with the [proper configuration](https://github.com/robotology/whole-body-controllers/blob/master/doc/How-to-setup-the-robot-for-wbc-experiments.md). 
+  To verify if the estimator is running, you can check weather data are streamed for the end-effectors to be used as sensors (e.g. `/wholeBodyDynamics/right_arm/endEffectorWrench:o`).
```
Comment (Yeshasvitvs) [inline doc/How-to-run-iCub-as-wearable-source.md]: **weather** -------> **whether**

File: doc/How-to-run-iCub-as-wearable-source.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+- In the [configuration file](https://github.com/robotology/wearables/blob/master/app/xml/ICubWearableDevice.xml), the following parameters are set propery:
```
Comment (Yeshasvitvs) [inline doc/How-to-run-iCub-as-wearable-source.md]: **propery** ---------> **properly**

File: doc/How-to-run-iCub-as-wearable-source.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  - `ft-sensors`: wrenche measurements that have to be exposed as wearable data. Those measurements should be provide by `whole-body-dynamics` estimator, or directly from sensors measurement.
```
Comment (Yeshasvitvs) [inline doc/How-to-run-iCub-as-wearable-source.md]: **provide ** --------> **provided**

**Comment by Yeshasvitvs** [comment]
>> /conf folder is removed since configuration files are stored in /app

@lrapetti for now we can proceed this ;) but soon we will do a cleanup and be consistent on both [human-dynamics-estimation](https://github.com/robotology/human-dynamics-estimation) (currently we use `conf` directory) and [wearables](https://github.com/robotology/wearables) repos

**Comment by lrapetti** [comment]
Thanks @Yeshasvitvs, I have fixed the typo in the documentation (https://github.com/robotology/wearables/pull/72/commits/df0e181228e17eea9b9a3cdfe6660815be745491)

**Comment by Yeshasvitvs** [comment]
Thanks @lrapetti 
Merging

**Pull request merged**